resources/lang/en-US/admin/consumables/message.php

```diff
@@ -26,6 +27,8 @@
         'success' 		=> 'Consumable checked out successfully.',
         'user_does_not_exist' => 'That user is invalid. Please try again.',
          'unavailable'      => 'There are not enough consumables for this checkout. Please check the quantity left. ',
+         'checkout_error' => 'Something went wrong with your Checkout',
+         'large_quantity_error' => '10,000 is the max quantity per checkout.',
```

Comment:
We don't generally want to hardcode limits into translations, since those limits can change and would then require re-translations.